js/common/modules/@arangodb/foxx/check-args.js

```diff
@@ -207,64 +242,90 @@ exports.validateStatus = function (value) {
   return {value: status, error: null};
 };
 
-exports.validateSchema = function (value) {
-  if (value) {
-    if (value.isJoi || typeof value.validate === 'function') {
-      return {value, error: null};
+exports.validateSchema = function (schema = true) {
+  if (schema === null) {
```

Comment:
Conventionally we allow passing `null` as a schema to mean "no value allowed" so we need to convert this.